operators/pkg/controller/elasticsearch/volume/volume.go

```diff
@@ -33,6 +33,9 @@ const (
 	UnicastHostsFile            = "unicast_hosts.txt"
 
 	ProcessManagerEmptyDirMountPath = "/mnt/elastic-internal/process-manager"
+
+	ElasticsearchDataVolumeName = "elasticsearch-data"
```

Comment:
We should probably update all samples and the quickstart doc to use this name in the VolumeClaimTemplates section.

Reply:
Already updated that in the "normalization" PR.